### DISCUSSION TITLE
Where is gadgetron_ismrmrd_client?

### DISCUSSION THREAD

**Discussion by markymarkymark** [gadgetron/gadgetron#1273] [General]
I just downloaded and launched _gadgetron/ubuntu_2204_base_

I was able to convert a Siemens rawdata file to .h5 with _siemens_to_ismrmrd_

But there is no executable for _gadgetron_ismrmrd_client_

Where is it? Do I need to build it? I gave the gadetron distro, but could use some help on where to install it and build.

I used earlier versions of gadgetron which came with gadgetron_ismrmrd_client installed. ???

**Comment by rajramasawmy** [comment]
I believe if you install the conda package, it will include the client
https://gadgetron.readthedocs.io/en/latest/obtaining.html#installing-in-conda-environment
